### ISSUE TITLE
Callout as table is placed at end of text 

### ISSUE THREAD

**Issue by cristianmtr** [mvdkwast/obsidian-copy-as-html#24]
**Describe the bug**

When exporting a text with a callout and with the option "Render some elements as tables" on, the callout is pasted at the end, instead of its original place.

**To Reproduce**

```markdown
Text first

> [!info] A callout
> Callout text.

Second text.
```

Export looks like

```
Text first
Second text.
A callout
Callout text.

```

**Expected behavior**

I'd expect

```
Text first
A callout
Callout text.
Second text.
```

Desktop, Windows 10, Obsidian 1.1.16, plugin version 0.4.0

**Comment by mvdkwast** [comment]
Hi @cristianmtr, thanks for the report and test-case, I can reproduce the issue.

**Comment by mvdkwast** [comment]
Fixed in release 0.4.2
